db.js

```diff
@@ -149,7 +149,20 @@ var friends = (function () {
 
 var users = (function () {
   return {
-    get: function (userId) {
+    getUserByName: function (user) {
+      var sql = "SELECT * FROM users WHERE name = ?";
```

Comment:
You should do a more generic search. Use something akin to `WHERE name LIKE '%name%'`. Look up how to use LIKE and wildcard syntax in SQL